deploy/sre-prometheus/100-sre-pv.PrometheusRule.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-pv
+    role: alert-rules
+  name: sre-pv
+  namespace: openshift-monitoring
+spec:
+  # https://issues.redhat.com/browse/OSD-3085
+  # Existing OCP alerts covering core namespaces:
+  # * KubePersistentVolumeUsageCritical
+  # * KubePersistentVolumeFullInFourDays
+  # Additional alerts for customers:
+  # * KubePersistentVolumeUsageCriticalCustomer
+  # * KubePersistentVolumeFullInFourDaysCustomer
+  # Additional alerts for non-RHMI layered products:
+  # * KubePersistentVolumeUsageCriticalLayeredProduct
+  # * KubePersistentVolumeFullInFourDaysLayeredProduct
+  groups:
+  - name: sre-pv-customer
+    rules:
+      # This matches with the default KubePersistentVolumeUsageCritical; this rule inverts the regex match.
+    - alert: KubePersistentVolumeUsageCriticalCustomer
+      expr: |
+        100
+        * kubelet_volume_stats_available_bytes{job="kubelet",namespace!~"(^openshift-.*|^kube-.*|^default$|^redhat-.*)"}
+        / kubelet_volume_stats_capacity_bytes{job="kubelet",namespace!~"(^openshift-.*|^kube-.*|^default$|^redhat-.*)"}
+        < 3
+      labels:
+        severity: critical
+        namespace: dedicated-admin
```

Comment:
Why are we overriding the label here? Shouldn't this come from the PV that is having high usage? (same question for all alerts)

Reply:
Initially thought we didn't get a namespace in the result to work with but in poking it again I see we do.  I'll update it.